elasticdl/python/master/master.py

```diff
@@ -76,6 +63,30 @@ def prepare(self):
         self._master_server.start()
         logger.info("Master RPC server started")
 
+    def _set_command_in_pod_manager(self):
+        if self.elasticdl_job_service:
+            command = self.elasticdl_job_service.get_ps_worker_command()
+            self.pod_manager.set_up(
+                worker_command=command,
+                worker_args=self.elasticdl_job_service.get_worker_args(
+                    self._args
+                ),
+                ps_command=command,
+                ps_args=self.elasticdl_job_service.get_ps_args(self._args),
+            )
+        elif self._args.job_command:
+            self.pod_manager.set_up(
+                worker_command=["/bin/bash"],
```

Comment:
I'm not sure that whether the customized job_command always starts with "/bin/bash -c"

Reply:
I think the job command should be a shell command and can use "/bin/bash -c" to execute the shell command.  What's more, the command in the container must be `["/bin/bash", "-c", "python -m xxxx"]` not `["/bin/bash -c python -m xxxx"]`. Because the first string the command list must be the path of an executable file.